portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -78,15 +84,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String email = request.getParameter("email").trim();
     if (email.isEmpty()) email = "@";
 
-    SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy");
-
     Entity commentEntity = new Entity("Comment");
     commentEntity.setProperty("author", author);
     commentEntity.setProperty("email", email);
     commentEntity.setProperty("comment", comment);
-    commentEntity.setProperty("date", formatter.format(new Date()));
+    // Subtract 4 hours to show EST time because local is not Eastern
+    commentEntity.setProperty("date", LocalDateTime.now().minusHours(4).format(DateTimeFormatter.ofPattern("yy/MM/dd HH:mm:ss")));
```

Comment:
I wonder if this will make it show some obscure time for us east coast folks. Is there another way you can ensure east coast time?

Reply:
Yeah this was a quick hack to make it show east coast time, but it does work as is I think. Will look into a better way to set eastern time though.

Reply:
I think If I were to deploy this App, it wouldn't show east coast for me, but I could be wrong.

Reply:
I think you're probably right. I think the ideal solution would be that no matter what time is stored in Datastore, the user sees timestamps based on their local time. Maybe the solution is to store an isostring for date/time and actually parse it on the frontend to set it to local time.

Reply:
Yeah just looked into it, what I'll do is I'll save the epoch seconds from server and when I call server data from client, I'll create a new JS Datatime object with the epoch seconds in local time. That should work.